.eslintrc

```diff
@@ -10,7 +10,7 @@
   "plugins": ["jsx-a11y", "header"],
   "rules": {
     "block-scoped-var": "error",
-    "complexity": ["error", {"max": 20}],
+    "complexity": ["error", {"max": 21}],
```

Comment:
https://github.com/google/web-stories-wp/pull/575/files#diff-2a6295bed180679fbd6e9ec9fef82cbdR109

Reply:
Instead of changing this globally, please change it only for the code in question.

You can increase the max allowed complexity for a specific file by adding `/* eslint complexity: ["error", { "max": 30 }] */` at the top.

However, I'd say at this point it might be reasonable to split out reducers into their own separate files, like in the story reducer.

Reply:
Perfect! I will go in this way. Thank you for prompt feedback @swissspidy